pkg/network/multus.go

```diff
@@ -24,7 +24,7 @@ func RenderMultus(conf *operv1.NetworkSpec, manifestDir string) ([]*uns.Unstruct
 
 	var err error
 	out := []*uns.Unstructured{}
-	objs := []*uns.Unstructured{}
+	var objs []*uns.Unstructured
```

Comment:
as in the other file, see if these declarations in this file can just be removed entirely

Reply:
fixed in updated patchset